Example/SettingsTableViewController.swift

```diff
@@ -4,27 +4,40 @@ class SettingsTableViewController: UITableViewController {
 
     @IBOutlet weak var textFieldAppID: UITextField!
     @IBOutlet weak var textFieldSubKey: UITextField!
-    @IBOutlet weak var textFieldURL: UITextField!
-    @IBOutlet weak var textFieldVersion: UITextField!
+    @IBOutlet weak var textFieldURL: UITextField?
+    @IBOutlet weak var textFieldVersion: UITextField?
```

Comment:
I don't think we are going to use these Custom runtime config params anymore. What do you think?